frappe/model/rename_doc.py

```diff
@@ -509,6 +509,8 @@ def get_link_fields(doctype: str) -> list[dict]:
 		)
 		if virtual_doctypes:
 			custom_fields = custom_fields.where(cf.dt.notin(virtual_doctypes))
+		if frappe.db.has_column("Custom Field", "is_virtual"):
```

Comment:
You can assume the existence of this column, it's been there since years. Please remove this check and merge the condition to where it's appropriate.

Reply:
I have removed the checks for an 'is_virtual' column for both standard and custom fields from the method.